hack/bicep-types-radius/src/autorest.bicep/src/type-generator.ts

```diff
@@ -142,66 +138,74 @@ export function generateTypes(host: AutorestExtensionHost, definition: ProviderD
       }
 
       const { descriptor, bodyType } = output;
+      let resourceTypeFunctions: Record<string, ResourceTypeFunction> = {};
+      
+      let actions = resourceFunctionType[fullyQualifiedType];
```

Comment:
We are now tracking functions per resource instead of overall. So each function will be in the types file associated with that resource's namespace.